@here/harp-mapview/lib/ThemeLoader.ts

```diff
@@ -82,6 +82,7 @@ export class ThemeLoader {
         }
 
         if (theme.styles) {
+            let globalThemeIndex = 0;
```

Comment:
remove ?

Reply:
Thanks for catching this one! Was a leftover.